application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -50,16 +58,36 @@ public void onReady(@NotNull ReadyEvent event) {
                             new CommandData(command.getCommandName(), command.getDescription())))
                         .queue();
                 }
+            }, throwable -> {
+                handleErrors(throwable, guild);
             });
         });
     }
 
+    private static void handleErrors(Throwable throwable, Guild guild) {
+        new ErrorHandler().handle(ErrorResponse.MISSING_ACCESS, errorResponseException -> {
+            // * Hard to read, there's no other accurate way to get a channel you've the write
+            // * permission in.
+            guild.getTextChannelCache().stream().filter(textChannel -> {
```

Comment:
This can be a utility method `Optional<TextChannel> getTextChannelWithWritePermissions(Guild)` to aid readability

Reply:
Creating something like

ChannelUtil#getChannelsWithPermissions(Guild, Permissions) seems fair
But this should be done within another PR

Reply:
Ok. I have already created one JDAUtils. I will move it in my PR instead.

Reply:
Don't remember JDAUtils having anything relating, and JDAUtils is extremely outdated

Reply:
🤦 thats something I have written and will pull out your piece.

Reply:
Ah, thought you were talking about JDA utlities